src/identity/Identity.ts

```diff
@@ -100,20 +192,101 @@ export default class Identity extends PublicIdentity {
 
   private readonly signKeyringPair: KeyringPair
   private readonly boxKeyPair: BoxKeyPair
+  /**
+   * @description Creates a new instance of public identity.
+   * @returns An `address` which identity's a specific account on the chain.
+   * @returns  A `boxPublicKeyAsHex` public half of the cryptographic key pair in hexadecimal
+   * @example
+   * ```javascript
+   *
+   * const alice = Kilt.Identity.buildFromMnemonic();
+   *
+   * // The identity is able to call the getPublicIdentity()
+   *
+   * alice.getPublicIdentity()
```

Comment:
There should be an assignment (see my comment on `didDocument`).

Reply:
Updated :)